src/components/InventoryHostStaleness/HostStalenessCard.js

```diff
@@ -68,13 +64,61 @@ const HostStalenessCard = () => {
         filterKey !== 'id' &&
         (apiData[filterKey] = daysToSecondsConversion(newFormValues[filterKey]))
     );
+
     //system_default means the account has no record, therefor, post for new instance of record.
     if (filter.id === 'system_default') {
-      axios.post(`${INVENTORY_API_BASE}/account/staleness`, apiData);
+      axios
```

Comment:
These API funcs are better placed under src/api.js. Also, take a look at the `instance` variable available there. It's an axios instance that you can already use to build and make the requests with.

Reply:
The instance version isnt resolving the promise in a way that the .catch will read. It will always hit the .then side of things. I've sent up a compromise where I move it to the api file but still use the useAxiosWithPlatformInterceptors hook